lib/opensusebasetest.pm

```diff
@@ -197,6 +197,8 @@ sub select_bootmenu_more {
     else {
         send_key_until_needlematch($tag, 'down', 10, 5);
     }
+    # Hyper-V defaults to 1280x1024, we need to fix it here
+    type_string 'video=hyperv_fb:1024x768' if check_var('VIRSH_VMM_FAMILY', 'hyperv');
```

Comment:
Hm, commonly we set the video resolution boot parameters within lib/bootloader_setup.pm . Could you use a helper function from there?

Reply:
Now there's a more generic approach to setting GFX.